meson.build

```diff
@@ -394,7 +390,17 @@ if sys_windows or get_option('lua-interpreter') == 'lua'
       break
     endif
   endforeach
+  if not lua.found()
+    lua = dependency('lua', fallback : ['lua', 'lua_dep'])
+    if not lua.found()
+      error('Lua not found')
+    endif
+  endif
 else
+  if sys_windows 
+    warning('There is no "' + get_option('lua-interpreter')
+            + '" binding option for Windows, backing up to "lua"')
+  endif
```

Comment:
Where is this backup happening? If we want to do this backup thing, why not let how it was before this PR (lines 384~388)?